src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/help-yaml.html

```diff
@@ -0,0 +1,26 @@
+<!--
+  ~ Copyright (C) 2015 Original Authors
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~         http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+The raw yaml of a Pod API Object.  Any pod fields set with non-empty values via the configuration fields above will take precedence
+as they are merged with this yaml.
+
+Fragments of a full Pod API Object yaml representation are allowed, but the yaml fragment must start with:
```

Comment:
must it ?

Reply:
Nope, you are right.  Just need enough context ... i.e. 

```
spec:
   tolerations:
      - key: CriticalAddonsOnly
        operator: Exists
```

works, but 

```
   tolerations:
      - key: CriticalAddonsOnly
        operator: Exists
```

Does not since the code puts in a empty spec if one is not specified.  I didn't try the middle ground before.

I'll update the help for this as well.

thanks